src/Faker/Provider/pt_BR/Person.php

```diff
@@ -99,4 +99,59 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+
+    /**
+     * A random CPF number.
+     * @link http://en.wikipedia.org/wiki/Cadastro_de_Pessoas_F%C3%ADsicas#Validation
+     * @param bool $formatted If the number should have dots/dashes or not.
+     * @return string
+     */
+    public function cpf($formatted = true)
+    {
+        $n = $this->generator->numerify('#########');
+        $n .= $this->verifierDigit($n);
+        $n .= $this->verifierDigit($n);
+
+        return $formatted? vsprintf('%d%d%d.%d%d%d.%d%d%d-%d%d', str_split($n)) : $n;
+    }
+
+    /**
+     * A random CNPJ number.
+     * @link http://en.wikipedia.org/wiki/CNPJ
+     * @link http://pt.wikipedia.org/wiki/CNPJ#Algoritmo_de_Valida.C3.A7.C3.A3o
+     * @param bool $formatted If the number should have dots/slashes/dashes or not.
+     * @return string
+     */
+    public function cnpj($formatted = true)
```

Comment:
I'm not really sure if we need to create a whole new provider just to generate company documents. Even more if you consider that's a juridic person document, same as the CPF for a physical person.